### PR TITLE
Responsive Improvements

### PR DESCRIPTION
**Pull request by jaimerson**
Here's the updated version of the reponsive improvements. I have found it better to start over than to undo the other line of work, so I'm opening a new PR and closing #59.
The analytics fix is at [kanaui#12](https://github.com/killbill/killbill-analytics-ui/pull/12)
Here is how it is looking on ipad landscape:
image

Please let me know if there is anything else while I try to fix portrait mode.

### CODE REVIEW

**Comment by jaimerson** [comment]
image
Ipad portrait

**Comment by sbrossie** [comment]
@jaimerson We will take a look and get back to you. Hopefully by mid week.

**Comment by jaimerson** [comment]
Hi,
Have you had the time to review these changes?

Em seg, 2 de mai de 2016 15:40, Stéphane Brossier <notifications@github.com>
escreveu:

**Comment by sbrossie** [comment]
@jaimerson We reinstalled your latest code on http://ec2-52-34-80-100.us-west-2.compute.amazonaws.com:3100

Unfortunately, we still don't see very much improvement and things that used to work on desktop are now broken:

* Top menu: spans across two lines now. Also note that you hardcoded the `plugin` menu. This one comes populated from the various rails engine that are mounted.  You need to test with the full UI (standalone), since it injects HTML/JS on the fly when mounting the engines.
* Table results: Still don't see any changes (iPad landscape). 
* Analytics: refresh button is now too large, even on Desktop, as it spans the whole column

I am wondering what is the best way to converge on that task.  Do you think a call would help, or do you also understand the issues we are raising?

**Comment by jaimerson** [comment]
Is the top menu spanning across two lines on desktop? Or it it just on iPad
portrait? That was a quick way to fix.

I have been working with the standalone, but I wasn't able to enable the
plugins (I'm using the killbill docker container). I'll try again to follow
the tutorial, or maybe you can provide one of those populated servers, so
that I can also solve the results table issue.

About there being no improvement, now I'm not sure what you expect of the
interface. We can make a call this week, if you'd like, to figure this out.

Em sáb, 7 de mai de 2016 20:38, Stéphane Brossier <notifications@github.com>
escreveu:

**Comment by jaimerson** [comment]
Search results on ipad portrait mode:
image

**Comment by jaimerson** [comment]
Update: confirmed that menu spans across two lines on desktop < 1300ish px; fixing.
Also, I could not reproduce the on-the-fly generation of plugins links on navbar, is there anything other than what's in [the tutorial](https://github.com/killbill/killbill-cloud/tree/master/docker) that I need to do? should I just remove the hardcoded one?

**Comment by jaimerson** [comment]
Working on a flex-box based solution for the navbar (using https://github.com/10up/flexibility for old ie support). You guys ok with that?

**Comment by sbrossie** [comment]
@jaimerson Ok, let's plan for a call, i think that will make things easier. Pierre is traveling and back on thursday, so i'll have to check with him first. Will get back to you.

**Comment by jaimerson** [comment]
Alright. In the meantime, I'll try and fix the remaining issues.

Em seg, 9 de mai de 2016 17:30, Stéphane Brossier <notifications@github.com>
escreveu:

**Comment by sbrossie** [comment]
@jaimerson Ok great. What about a call on  friday 9am PDT ?

**Comment by jaimerson** [comment]
Works for me

Em seg, 9 de mai de 2016 19:14, Stéphane Brossier <notifications@github.com>
escreveu:

**Comment by jaimerson** [comment]
image
image
image

File: app/views/kaui/layouts/kaui_header.html.erb [hunk 1]
```diff
@@ -8,6 +8,8 @@
   <%= javascript_include_tag 'application' %>
   <%= csrf_meta_tags %>
 
+  <script src="https://cdnjs.cloudflare.com/ajax/libs/flexibility/1.0.6/flexibility.js"></script>
```
Comment (pierre) [inline app/views/kaui/layouts/kaui_header.html.erb]: Is there a packaged version we could bundle? When Kaui is deployed to private datacenters, connectivity to the outside is most likely restricted.

File: app/assets/stylesheets/kaui/kaui.less [hunk 2]
```diff
@@ -1,3 +1,5 @@
+@import "twitter/bootstrap/bootstrap";
```
Comment (pierre) [inline app/assets/stylesheets/kaui/kaui.less]: Is that necessary? It should already be imported (see https://github.com/killbill/killbill-admin-ui/blob/c4fb6993d99c736ffde779f26353eaff60d256a6/app/assets/stylesheets/application.css#L9).

Comment (jaimerson) [inline app/assets/stylesheets/kaui/kaui.less]: I had problems with the mixins using the other import. But I removed the duplication leaving only this one (and renaming `bootstrap_and_overrides` to `bootstrap_overrides`)

**Comment by jaimerson** [comment]
All fixed :shipit: :rocket:

**Comment by sbrossie** [comment]
@jaimerson Just took a look as well. This is better, but not quite there:

* On iPad landscape, the top menu is not right:
  * The `Kill Bill` logo, search bar and `create account` are too close to each other and crammed
  * The little grey search magnifying glass sits kind of randomly in the middle of the screen
  * The arrow with text `demo_ro/demo` and `SIGN OUT` is too close to the gear

* On web browser the analytics screen has not been fixed (attached a screen shot below). `End date` is too wide compared to what it was before and `refresh` button now appears below the box

And then there is a iPad portrait (which we said we would delay), and where all tables are broken.

**Comment by sbrossie** [comment]
Also adding the screenshot for the broken search:

image

**Pull request closed without merging**